contracts/vendor/Timelock.sol

```diff
@@ -99,15 +91,17 @@ contract BridgeTimelock is ITimelock {
             callData = abi.encodePacked(bytes4(keccak256(bytes(signature))), data);
         }
 
+        // solium-disable-next-line security/no-call-value
```

Comment:
I assume we're just trying to minimize the diff? I think we could get rid of these solium hints...

Reply:
sure, will do